src/js/player.js

```diff
@@ -2860,38 +2853,32 @@ class Player extends Component {
    * @fires Player#fullscreenchange
    */
   requestFullscreen(fullscreenOptions) {
-    const PromiseClass = this.options_.Promise || window.Promise;
+    const self = this;
 
-    if (PromiseClass) {
-      const self = this;
-
-      return new PromiseClass((resolve, reject) => {
-        function offHandler() {
-          self.off('fullscreenerror', errorHandler);
-          self.off('fullscreenchange', changeHandler);
-        }
-        function changeHandler() {
-          offHandler();
-          resolve();
-        }
-        function errorHandler(e, err) {
-          offHandler();
-          reject(err);
-        }
-
-        self.one('fullscreenchange', changeHandler);
-        self.one('fullscreenerror', errorHandler);
+    return new Promise((resolve, reject) => {
+      function offHandler() {
+        self.off('fullscreenerror', errorHandler);
+        self.off('fullscreenchange', changeHandler);
+      }
+      function changeHandler() {
+        offHandler();
+        resolve();
```

Comment:
Are these newly not covered by tests or were they always not covered by tests?

Reply:
They were always not covered. This PR technically increased project coverage by [.05%](https://app.codecov.io/gh/videojs/video.js/compare/7715/diff).